clients/da-vinci-client/src/main/java/com/linkedin/davinci/consumer/InternalLocalBootstrappingVeniceChangelogConsumer.java

```diff
@@ -195,14 +225,25 @@ public void onRecordReceived(byte[] key, byte[] value) {
                         getTopicPartition(state.getKey()),
                         0,
                         0,
-                        value.length * 8);
+                        value.length * 8,
+                        false);
                 resultSet.add(record);
               }
 
               @Override
               public void onCompletion() {
                 // Update the map so that we're no longer in bootstrap mode
                 state.getValue().bootstrapState = PollState.CONSUMING;
+                // Add a dummy record to mark the end of the bootstrap.
```

Comment:
Just to make sure, this as I read it will send a dummy record with endOfBootStrap set to true for one partition, NOT for the end of all local state.  Is that ok?  I'm struggling to recall what the ask was for this.

Reply:
That's a good point. Changed to check the bootstrap completed count and will only send this dummy record when all partitions have completed.